facp.md

```diff
@@ -0,0 +1,82 @@
+# FACP
+
+The FADT, of which the signiture is FACP, refers to the Fixed ACPI Description Table, and is pointed by the RSDT.
+The RSDT has entries that point to ACPI, FADT and more.
+The entries are 4 in number, and each of them are pointing to the tables.
+FACP is one of them, and could be found by strncmp.
```

Comment:
Mention the fact that they are string not null terminated, and since we are in osdev world, strncmp maybe is not available, so probably better to say something; 

"And could be found by a strncmp like function"